bitbucket-pipelines.yml

```diff
@@ -0,0 +1,53 @@
+image: 
+  name: aligent/shopify-cli:latest
+
+definitions:
+  steps:
+    - step: &theme-check
+        name: Theme Check
+        script:
+          - echo "Starting THEME CHECK..."
+          - ruby -v && node -v && npm -v && shopify version
+          - find . -exec ls -ld --time-style=long-iso {} \; | awk '{print $6, $7, $8, $9}'
```

Comment:
This line probably needs a comment, I'm not sure what it's doing or why :sweat_smile:

Reply:
Updated